workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -37,6 +37,14 @@ knitr::opts_chunk$set(
 )
 ```
 
+```{r cutoffs, include=FALSE, cache=TRUE}
```

Comment:
I'm unsure about having a global alpha and lfc threshold that we use for everything, as we sometimes run two versions (say, one with lfc.thresh=0 and another with lfc.thresh=1). @mitraak could you weigh in on this?

Reply:
I have been leaning towards this lately. Multiple lfc thresholds are fine during the exploratory phase, but eventually we usually choose one unless the collaborators want that tweaked.

Reply:
I like the idea of using the `metadata()` function to unify the lfc and fdr thresholds in the helper functions. That will definitely clean up the code.